src/state/transactions/updater.tsx

```diff
@@ -53,6 +54,7 @@ export default function Updater() {
               )
               dispatch(finalizeOrderCancellation())
               dispatch(finalizeOrderPlacement())
+              dispatch(pullOrderbookData())
```

Comment:
I like this thinking, though, currently, this is not reliably working.

Since users and the backend node do not connect to the same node, this syncing might actually fail, as the backend has not yet processed the new blockchain event.
 I also don't think that it is needed, as the user order gets automatically updated into the orderbook data, even before the order is placed. You can see it live during order placement phase as the organe orders